src/main/java/com/yahoo/bullet/rest/query/PubSubReader.java

```diff
@@ -18,6 +22,8 @@
     private ConcurrentMap<String, QueryHandler> requestQueue;
     private Thread readerThread;
     private int sleepTimeMS;
+    private static final Set<Metadata.Signal> FINISHED = new HashSet<>(
```

Comment:
move new HashSet and combine with next line?

Reply:
Done.